### PR TITLE
Add tests for miscellaneous checks

### PR DESCRIPTION
**Pull request by philsmart**

### CODE REVIEW

File: tests/xml/misc/bad_misc_org_displayname_line_break.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            Location="https://idp.example.org/idp/profile/SAML2/POST/SSO"/>       
+        <SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST-SimpleSign"
+            Location="https://idp.example.org/idp/profile/SAML2/POST-SimpleSign/SSO"/>
+        <SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect"
+            Location="https://idp.example.org/idp/profile/SAML2/Redirect/SSO"/>
+    </IDPSSODescriptor>
+    <Organization>
+        <OrganizationName xml:lang="en">An Example Organization</OrganizationName>
+        <OrganizationDisplayName xml:lang="en">An Example Organization. Line break now...
+
```
[the remark this answers is not in the document]
Comment (philsmart) [reply tests/xml/misc/bad_misc_org_displayname_line_break.xml:96]: When I make this change, would you prefer I force-push a clean PR. Or add the commit for the changes to the end?

[the remark this answers is not in the document]
Comment (philsmart) [reply tests/xml/misc/bad_misc_org_displayname_line_break.xml:96]: I would assume a clean PR, but just checking

[the remark this answers is not in the document]
Comment (iay) [reply tests/xml/misc/bad_misc_org_displayname_line_break.xml:96]: You can do either, it's up to you. It's probably easier to verify changes requested as part of a review if they start out in another commit. If you go that route, though, you would squash everything into a single one when landing the PR; changes for review don't belong in the main line once you're done.

**Comment by philsmart** [comment]
Added the second example with two lines, the second after the line break

**Review by iay — APPROVED** [review]

**Comment by philsmart** [comment]
Squashed and pushed

**Pull request merged**